src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -18,7 +18,6 @@ namespace Neo.SmartContract.Native.Tokens
     public sealed class NeoToken : Nep17Token<NeoToken.NeoAccountState>
     {
         public override int Id => -1;
-        public override string Name => "NEO";
```

Comment:
It will change the name of `Neo` to `NeoToken`

Reply:
Yes. It is the contract name. Usually we use symbol.